core-services/prow/02_config/openshift/api/_prowconfig.yaml

```diff
@@ -99,7 +159,10 @@ tide:
     - release-4.9
     labels:
     - approved
+    - docs-approved
     - lgtm
+    - px-approved
+    - qe-approved
```

Comment:
FWIW we never added these approvals to the complement query. These criteria will only apply on non-master, non-relesae branches which are usually just feature branches and formal approvals are too heavy handed for it.

Reply:
hmm, we did in at least some places:

https://github.com/openshift/release/blob/8643b11ed69c67035aa2fb9e2454112ff8a352ff/core-services/prow/02_config/openshift/builder/_prowconfig.yaml#L133-L165

but apparently not consistently.  i'm happy to remove it from this one.

Reply:
@petr-muller updated